lib/common.sh

```diff
@@ -93,11 +93,14 @@ FILESYSTEM=${FILESYSTEM:="/"}
 # BMO_RUN_LOCAL : run the Baremetal Operator locally (not in Kubernetes cluster)
 # CAPM3_RUN_LOCAL : run the Cluster API Provider Metal3 locally
 
-export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
 export CAPM3_VERSION="${CAPM3_VERSION:-"v1alpha5"}"
-CAPM3_VERSION_LIST="v1alpha4 v1alpha5"
-if ! echo "${CAPM3_VERSION_LIST}" | grep -wq "${CAPM3_VERSION}"; then
-  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : ${CAPM3_VERSION_LIST}"
+
+if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
+elif [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha3"}"
+else
+  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : {v1alpha4, v1alpha5}"
```

Comment:
instead of hardcoding API versions here, IMO it would be better to keep them in the original `CAPM3_VERSION_LIST` var and use it the way it was used originally.

Reply:
As we have now the if else condition to check the CAPI version needed the last else can throw the error if now version was matched restoring the original 
```
CAPM3_VERSION_LIST="v1alpha4 v1alpha5"
if ! echo "${CAPM3_VERSION_LIST}" | grep -wq "${CAPM3_VERSION}"; then
  echo "Invalid CAPM3 version : ${CAPM3_VERSION}. Not in : ${CAPM3_VERSION_LIST}"
  ```
  Seems duplicate checking ?